.vale/fixtures/RedHat/ObviousTerms/.vale.ini

```diff
@@ -0,0 +1,5 @@
+; Vale configuration file to test the `ObviousTerms` rule
+StylesPath = ../../../styles
+MinAlertLevel = suggestion
+[*.adoc]
+RedHat.ObviousTerms = YES
```

Comment:
I don't like this rule in RedHat because it's highlighting AsciiDoc formatting, and this Vale rule set should be agnostic to formatting.

Reply:
I agree. Problem is that Vale does not accept "strong" as a scope because it has been deprecated in favor of "raw". Maybe adding "field" is the way to go.